tasks/build.py

```diff
@@ -389,3 +392,25 @@ def submit_build_jobs(pr, event_info):
         create_metadata(job, repo_name, pr, job_id)
         # report submitted jobs (incl architecture, ...)
         create_pr_comments(job, job_id, app_name, job_comment, pr, repo_name, gh, symlink)
+
+
+def build_built_artefacts(pr, event_info):
```

Comment:
Rework this as `check_build_permission` which just returns `False` or `True`, and does logging, but does not call `submit_build_jobs`

And then in `handle_pull_request_labeled_event`:

```python
if check_build_permission(pr, event_info):
    submit_build_jobs(pr, event_info)
```

And make sure to add a docstring for `check_build_permission` here